ably.d.ts

```diff
@@ -912,11 +878,9 @@ declare namespace Types {
     /**
      * BEGIN CANONICAL DOCSTRING
      * The HTTP verb to use for the request, either `GET` or `POST`.
-     * END CANONICAL DOCSTRING
      *
-     * BEGIN CANONICAL DEFAULT INFO
-     * default .GET
-     * END CANONICAL DEFAULT INFO
+     * @defaultValue `.GET`
```

Comment:
This one should be `"GET"`

Reply:
Changed to <code>@defaultValue &#96;HTTPMethods.GET&#96;</code>.